corehq/apps/export/views/list.py

```diff
@@ -175,17 +174,252 @@ def _fmt_emailed_export_fileData(self, fileId, size, last_updated,
             'downloadUrl': download_url,
         }
 
-    def get_exports_list(self):
-        # Calls self.get_saved_exports and formats each item using self.fmt_export_data
-        saved_exports = self.get_saved_exports()
-        if toggles.EXPORT_OWNERSHIP.enabled(self.request.domain):
-            saved_exports = [
-                export for export in saved_exports
-                if export.can_view(self.request.couch_user.user_id)
-            ]
-        if self.is_deid:
-            saved_exports = [x for x in saved_exports if x.is_safe]
-        return list(map(self.fmt_export_data, saved_exports))
+
+class DailySavedExportListHelper(ExportListHelper):
+    @memoized
+    def get_saved_exports(self):
+        combined_exports = []
+        if self.permissions.has_form_export_permissions:
+            combined_exports.extend(get_form_exports_by_domain(self.domain,
+                                                               self.permissions.has_deid_view_permissions))
+        if self.permissions.has_case_export_permissions:
+            combined_exports.extend(get_case_exports_by_domain(self.domain,
+                                                               self.permissions.has_deid_view_permissions))
+        combined_exports = sorted(combined_exports, key=lambda x: x.name)
+        return [x for x in combined_exports if x.is_daily_saved_export and not x.export_format == "html"]
+
+    def _get_edit_export_class(self, model):
+        from corehq.apps.export.views.edit import EditFormDailySavedExportView, EditCaseDailySavedExportView
+        return {
+            "form": EditFormDailySavedExportView,
+            "case": EditCaseDailySavedExportView
+        }[model]
+
+    def fmt_export_data(self, export):
```

Comment:
Guessing these methods are too dissimilar to cleanly deduplicate?

Reply:
yeah, going to leave these methods as they are

Reply:
you were right about this - cleaned them up in https://github.com/dimagi/commcare-hq/pull/23071